### ISSUE TITLE
'Signing' orders appear when the app is opened in 2 browser's windows

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#2537]
Discussed in https://cowservices.slack.com/archives/C0361CDG8GP/p1684936701475189?thread_ts=1684859235.739219&cid=C0361CDG8GP

Related to #2525

1. Open the CoW Swap app in a browser tab
2. Connect to a Safe using WC
3. Open the Safe apps, search for CoW Swap dApp there
4. Open the  CoW Swap dApp inside the Safe
5. Start placing an order: press on the Swap button, but not sign the order in Safe
6. Go back to the  CoW Swap web-app opened in the browser tab (p.1)
7. Check Your orders table in the Limit orders page

**AR**: Signing order is displayed there
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/6c4c2d45-2ac8-4819-ac3e-571bde5e1edd)
**ER**: Signing order should not be displayed on the UI until an order is signed by a safe owner.

**Comment by anxolin** [comment]
@alfetopito we didn't check this one, right? 

You mentioned it could be joitai no syncing across tabs, but this using Redux
https://github.com/cowprotocol/cowswap/blob/3266be7ece02e7c8b43ed360c2d82e464eae3a92/src/legacy/state/orders/hooks.ts#L159

not sure how important it is, I will place in icebox, please decide if you want to bring it back to Good to have or work don't this

**Comment by alfetopito** [comment]
No, I did not get to check this one. I said [I would check after Safe Eth flow](https://cowservices.slack.com/archives/C0361CDG8GP/p1684938009899929?thread_ts=1684859235.739219&cid=C0361CDG8GP)

**Comment by elena-zh** [comment]
Closing as there is nothing to do with this issue atm
